src/command/new/mod.rs

```diff
@@ -0,0 +1,348 @@
+use crate::utils::client::StudioClientConfig;
+use ansi_term::Colour::Cyan;
+use ansi_term::Style;
+use camino::Utf8PathBuf;
+use console::Term;
+use dialoguer::Select;
+use saucer::Utf8PathBuf as PathBuf;
+use saucer::{clap, Parser};
+use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
+use std::fmt;
+use std::fs::read_dir;
+use std::io::Write;
+use std::str::FromStr;
+
+use crate::options::TemplateOpt;
+use crate::{anyhow, command::RoverOutput, error::RoverError, Result};
+
+#[derive(Debug, Clone, Serialize, Parser)]
+pub struct New {
+  #[clap(flatten)]
+  options: TemplateOpt,
+
+  /// Path to create template at
+  #[clap(name = "path")]
+  #[serde(skip_serializing)]
+  path: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Parser)]
+pub struct GithubTemplate {
+  pub id: String,
+  pub display: String,
+  pub language: String,
+  pub project_type: String,
```

Comment:
```suggestion
  pub language: ProjectLanguage,
  pub project_type: ProjectType,
```

these should store the enum types directly instead of converting them to `String`s